cgmml/common/evaluation/CV/evaluation.py

```diff
@@ -180,25 +188,40 @@ def process_height_prediction(depthmap_file: str, rgb_file: str, calibration_fil
     metadata_file = path + '/' + sys.argv[2]
 
     avg_err = 0
+    last_fail = 0
     output = []
     rejections = []
-    indata = filter_metadata(read_csv(metadata_file), is_standing)
-    size = len(indata)
-    for index in range(size):
+    indata = filter_metadata(read_csv(metadata_file), is_standing, False)
+    keys = indata.keys()
+    size = len(keys)
+    index = 0
+    for key in keys:
```

Comment:
If you increase an index in every loop iteration, this is how you can do it in Python:

```suggestion
    index = 0
    for key_index, key in enumerate(keys):
```